docs/pages/blog/turbo-1-8-0.mdx

```diff
@@ -0,0 +1,118 @@
+- [**Easier migrations**](#global-turbo): Automatically migrate to new versions of turbo with `npx @turbo/codemod migrate`.
+
+Update today by running `npx @turbo@codemod migrate`.
+
+## Workspace Configurations
+
+In workspace directories, you can now add a `turbo.json` to:
+
+- add tasks specific to that workspace
+- override configuration for tasks in workspace directories
+
+This will enable teams to scale ownership of their monorepos by moving away from global configuration with fine-grain contol over tasks in workspaces.
+
+For example, imagine your monorepo has a Next.js app and a SvelteKit app, and you want to use Turborepo to cache outputs of the `build` task. The Next.js `build` script creates a `.next` directory, whereas SvelteKit creates a `.svelte-kit` directory. Instead of adding both build directories in your root `outputs`, you can define the `outputs` key in the workspace instead:
+
+```jsonc filename="turbo.json"
+{
+	"pipeline": {
+		"build": {
+			"dependsOn": ["^codegen"],
+			// no need to define outputs here!
+		}
+	}
+}
+```
+
+```jsonc filename="apps/my-nextjs-app/turbo.json"
+{
+	"extends": ["//"],
+	"pipeline": {
+		"build": {
+			// dependsOn is inherited from root
+			"outputs": [".next/**"]
+		}
+	}
+}
+```
+
+```jsonc filename="apps/my-sveltekit-app/turbo.json"
+{
+	"extends": ["//"],
+	"pipeline": {
+		"build": {
+			// dependsOn is inherited from root
+			"outputs": [".svelte-kit/**"]
+		}
+	}
+}
+```
+
+Keys that **are** declared will replace the key from the root if those keys exist, overriding what is defined in your root configuration. Keys that **aren’t** declared are inherited from the root config.
+
+In the example above, `outputs` is customized for both apps, while `dependsOn` is configured by the root `turbo.json` and remains `"^codegen"`.
+The `extends` key in the Workspace Configurations enables workspace owners to use the best of the root `turbo.json` and customize the parts that makes their app different (The `"//"` sigil will look familiar if you’re used to [defining tasks to run from your root](/repo/docs/core-concepts/monorepos/running-tasks#running-tasks-from-the-root)).
+
+[Learn more in the docs](/repo/docs/core-concepts/monorepos/configuring-workspaces).
+
+## Automatic Workspace Scoping
+
+In [version 1.7](/blog/turbo-1-7-0), `turbo` became globally installable, giving you the power to run your tasks no matter where you are in your codebase. However, `turbo` would still run tasks from the root, running tasks that are in other workspaces that you may not have been intending.
+
+With 1.8, `turbo` will automatically detect the workspace you are in and generate [the `--filter` syntax](/repo/docs/reference/command-line-reference#--filter) to scope your task to your current workspace.
+
+As an example, if you are working in `apps/admin` and use the `turbo build` command, `turbo` will run `turbo build --filter=admin` under the hood, focusing on the workspace that you are working on.
```

Comment:
I feel like "working in" is a little ambiguous. I could interpret that as "my editor has open" versus "have as current directory in the terminal"

Reply:
Good call, fixed.